fixed a loading bug.

BobCN2017 opened the pull request:
in config file,like this.
- name: "tokenizer_jieba"
user_dicts: "./jieba_userdict"
so the dictionary name should be "user_dicts".

**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/RasaHQ/rasa_nlu?pullRequest=1380) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/RasaHQ/rasa_nlu?pullRequest=1380) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RasaHQ/rasa_nlu?pullRequest=1380) it.</sub>

tmbo commented:
I don't think it should be `user_dicts` - where did you get that from, is it wrong in the docs?

BobCN2017 commented:
> I don't think it should be `user_dicts` - where did you get that from, is it wrong in the docs?

I used rasa_nlu about three month ago,From that time,I used pipeline like this:
pipeline:
- name: "nlp_mitie"
  model: "data/total_word_feature_extractor_zh.dat"
- name: "tokenizer_jieba"
  user_dicts: "./jieba_userdict"
- name: "ner_mitie"
- name: "ner_synonyms"
- name: "intent_entity_featurizer_regex"
- name: "intent_featurizer_mitie"
- name: "intent_classifier_sklearn"

some days ago,I finded the user_dictionary not to be loaded.At that time ,I checked the code,finded the name is "dictionary_path",It couldn't load my dictionary.when I changed the name to "user_dicts",It worked.so I think this is a bug and request the pr.


because you insist that, now I checked the code and docs carefully again,I finded it's my mistake,In the doc,It defined like this:
pipeline:
- name: "tokenizer_jieba"
  dictionary_path: "path/to/custom/dictionary/dir"
  
I changed "user_dicts" to "dictionary_path",now it works.
sorry about this request.

tmbo commented:
No worries, glad everything is working now 👍
